### PR TITLE
Functions and block Documentation

### PR DESCRIPTION
**Pull request by tdruiva**

### CODE REVIEW

File: src/or/prop259.c [hunk 1]
```diff
@@ -95,17 +95,21 @@ static smartlist_t *bridges = NULL;
...
-/** INTERNET_LIKELY_DOWN_INTERNAL is the interval time of minutes that we should choose again **/
+/** The interval time of minutes that we should choose again **/
 const int INTERNET_LIKELY_DOWN_INTERNAL= 5;
 /** We restrict the number of guards that is going to be used more often **/
 const int PRIMARY_GUARDS_SIZE = 3;
 const int PRIMARY_GUARDS_SIZE_CONSTRAINED  = 1;
 const int PRIMARY_GUARDS_RETRY_INTERVAL = 3;
+/** We restrict the sample set lenght that is going to be used **/
+const int MINIMUM_FILTERED_SAMPLE_SIZE = 20;
+const int MAXIMUM_SAMPLE_SIZE_THRESHOLD = 1.03;
```
Comment (tcz001) [inline src/or/prop259.c]: I guess this shoule be a float?

Comment (tcz001) [inline src/or/prop259.c]: https://travis-ci.org/twstrike/tor_for_patching/jobs/137851543

Comment (tdruiva) [inline src/or/prop259.c]: :+1:

**Comment by tdruiva** [comment]
@asn-the-goblin-slayer I update with some comments, please check if them make sense hehe

**Comment by asn-d6** [comment]
Thank you! We are slowly starting to get back on this project. Will check your changes!

**Pull request merged**